src/Toolkit/V.php

```diff
@@ -422,6 +422,17 @@ public static function __callStatic(string $method, array $arguments): bool
                V::min($value, 2) === true;
     },
 
+    /**
+     * Checks for a valid hex color
+     */
+    'hexColor' => function ($value): bool {
+        if (!is_string($value) || $value === '') {
+            return false;
+        }
+
+        return (bool)mb_ereg_match('^#?([0-9a-fA-F]{3}|[0-9a-fA-F]{6})$', $value);
```

Comment:
I just wonder that should `#` be optional?

Reply:
I'm wondering the same. The main problem I see with this is that the options `#` wouldn't allow me to enforce one way of entering the hex code.